lib/order.rb

```diff
@@ -0,0 +1,35 @@
+class Order
+  attr_reader :order_dishes, :menu_items, :final_order
+
+
+  def initialize
+    @order_dishes = []
+    @menu_items = {
+      "pizza" => 10,
+      "burger" => 6,
+      "chicken wings" => 3,
+      "coke" => 1,
+      "coffee" => 2
+    }
+    @final_order = []
+  end
+
+  def select_dish(dish)
+    @menu_items.each_key do |key|
+    @order_dishes << dish if key == dish
+    end
+  end
+
+  def final_order_compile
```

Comment:
maybe refactor this method for readability and to have it only do one thing, and delegate some things away to other methods?